packages/foreman/rubygem-secure_headers/rubygem-secure_headers.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name secure_headers
 
 Name: rubygem-%{gem_name}
-Version: 6.3.4
+Version: 6.4.0
 Release: 1%{?dist}
 Summary: Manages application of security headers with many safe defaults.
 License: ASL 2.0
```

Comment:
I was looking into this and the actual license is MIT: https://github.com/github/secure_headers/commit/02d65f10d76484f51dac1f2f8090f942cd8d0a21